chart/templates/_commonChecks.tpl

```diff
@@ -32,49 +25,41 @@ Return common collectors for preflights and support-bundle
         - {{ template "carto.workspaceMigrations.image" . }}
         - {{ template "carto.workspaceSubscriber.image" . }}
         - {{ template "carto.workspaceWww.image" . }}
+        - {{ template "carto.tenantRequirementsChecker.image" . }}
   - runPod:
-      collectorName: support-run-health-on-maps-api
-      name: support-run-health-on-maps-api
+      collectorName: tenant-requirements-check
+      name: tenant-requirements-check
       namespace: {{ .Release.Namespace | quote }}
       timeout: 180s
       podSpec:
         containers:
-          - name: run-health
-            image: {{ template "carto.mapsApi.image" . }}
-            imagePullPolicy: IfNotPresent
-            command: ["bash"]
-            args: ["-exc", "npm run ready-to-run:built"]
+          - name: run-tenants-requirements-check
+            image: {{ template "carto.tenantRequirementsChecker.image" . }}
+            imagePullPolicy: {{ .Values.tenantRequirementsChecker.image.pullPolicy }}
             env:
-            {{- include "carto.replicated.commonChecks.customerValues" . | indent 12 }}
-            {{- include "carto.replicated.commonChecks.customerSecrets" . | indent 12 }}
+            {{- include "carto.replicated.tenantRequirementsChecker.customerValues" . | indent 12 }}
+            {{- include "carto.replicated.tenantRequirementsChecker.customerSecrets" . | indent 12 }}
 {{- end -}}
 
 {{/*
 Return common analyzers for preflights and support-bundle
 */}}
 {{- define "carto.replicated.commonChecks.analyzers" }}
-  - postgres:
-      checkName: PostgreSQL is available
-      collectorName: workspace-db
-      outcomes:
-        - fail:
-            when: connected == false
-            message: Cannot connect to PostgreSQL server
-        - pass:
-            when: connected == true
-            message: The PostgreSQL server is available
-  - postgres:
-      checkName: PostgreSQL must be v14.x or later
-      collectorName: workspace-db
+  {{- range list "Check_database_connection" "Check_database_encoding" "Check_user_has_right_permissions" "Check_database_version" }}
```

Comment:
What do you think of define it as a template and call for each checker?
Right now, looks so impersonal. Do you think that is going to be solved with the option to extract information from the container message?

Reply:
Yes, if you refer to the message that we are going to see in each check, we will see the same message that the tenant-requirements-checker container is logging, e.g:

`Postgres version is 13.12, but it should be at least 13.2`

`User permissions are correct`

`Postgres connection successful!`


and so on...